packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -242,14 +242,14 @@ mixin DebugOverflowIndicatorMixin on RenderObject {
     // TODO(jacobr): add the overflows in pixels as structured data so they can
     // be visualized in debugging tools.
     FlutterError.reportError(
-      FlutterErrorDetailsForRendering(
+      FlutterErrorDetails(
         exception: FlutterError('A $runtimeType overflowed by $overflowText.'),
         library: 'rendering library',
         context: ErrorDescription('during layout'),
-        renderObject: this,
         informationCollector: () sync* {
           if (debugCreator != null)
             yield DiagnosticsDebugCreator(debugCreator);
+          yield DiagnosticsProperty<RenderObject>('RenderObject', this, style: DiagnosticsTreeStyle.errorProperty);
```

Comment:
similarly, renderObject instead of `RenderObject`.

Reply:
for this one you should give the property a lower priority level as you don't want it to actually show up as that will make the error confusing to read. I think `level: DiagnosticsLevel.debug` is what you want. https://api.flutter.dev/flutter/foundation/DiagnosticLevel-class.html